js/libs/form.js

```diff
@@ -114,6 +114,41 @@ Fliplet().then(function() {
       };
     }
 
+    function getMatrixValue(value, field) {
+      var matrixOption = {};
+
+      if (!value || typeof value === 'object') {
+        return value;
+      }
+
+      if (value.indexOf('[') > -1 || value.indexOf(']') > -1) {
+        _.forEach(value.split(/\r?\n/), function(rawOption) {
+          if (rawOption) {
```

Comment:
Use early exit to reduce the amount of indentations

```suggestion
          if (!rawOption) {
            return;
          }
```